manifests/s/Salesforce/sfdx-cli/7.176.1.0/Salesforce.sfdx-cli.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Salesforce.sfdx-cli
+PackageVersion: 7.176.1.0

```

```suggestion
PackageVersion: 7.176.1
```
This must match Control Panel.